README.md

```diff
@@ -6,4 +6,245 @@
 
 `npm install reshuffle-quickbooks-connector`
 
-### Reshuffle Quickbooks Connector
\ No newline at end of file
+### Reshuffle Quickbooks Connector
+
+This package contains a [Reshuffle](https://github.com/reshufflehq/reshuffle)
+connector to connect [QuickBooks Online app APIs](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/most-commonly-used/account).
+This connector requires to define a [Reshuffle Datastore](https://dev.reshuffle.com/docs/persistency) in order to maintain the Quickbooks access and refresh tokens. The connector takes care of refreshing the access token.
+
+The following example exposes an endpoint to return the data of Bill after an Update action, the access and refresh tokens are stored in reshuffle DB and will be managed internally by the Reshuffle engine.
+
+```js
+const { Reshuffle, SQLStoreAdapter } = require('reshuffle')
+const { QuickbooksConnector } = require('reshuffle-quickbooks-connector')
+
+const app = new Reshuffle()
+
+const pool = new Pool({user: 'RESHUFFLE_DB_USER',
```

Comment:
can't we use a in memory default? sounds complex to have to provide a DB for just a token

Reply:
Here I just gave an example of using DB, it can be memory, file or DB. It's the user's choice which option to choose. 
I'll update the readme with the other options